hadoop-cloud/src/hadoop-3/test/resources/log4j2.properties

```diff
@@ -1,40 +0,0 @@
-#
```

Comment:
Why delete this file instead of moving it to the `test/resources` directory as other modules?